### DISCUSSION TITLE
Improving the Governance Model

### DISCUSSION THREAD

**Discussion by JaimeMagiera** [open-gitops/project#115] [Ideas]
At the Jun 15th 2022 meeting, several complexities and shortcomings of the governance model and election process began to surface while we discussed maintainer and chair nominations/elections. This discussion thread will be a place to highlight and discuss areas where the governance model and ensuing documentation can be improved.

**Comment by todaywasawesome** [comment]
* Process to remove maintainers
* Clarify maintainers as ultimate power (not chairs)
* Create easier way for people to become involved and join teams/committees

**Reply by nishadmehendale** [reply]
The process to remove maintainers

- My suggestion would be to finalise an inactive period depending on the role they are performing. 
- If a maintainer doesn't show up for X months then they should be considered inactive.
- Other maintainers will try to reach them once they reach X months and N attempts will be made. If there are no successful attempts, removal is approved in the next meeting.
- X and N can be determined by existing chairs/maintainers, I would say X as 3 months is a good number. N can be 3.

**Reply by christianh814** [reply]
> Process to remove maintainers

I am in favor of quantifying what "inactive" means. But what I want to be cognizant of those who contribute async. Someone can, potentially, be very active in github and/or slack.
